x/ibc/applications/transfer/types/msgs.go

```diff
@@ -64,9 +64,10 @@ func (msg MsgTransfer) ValidateBasic() error {
 	return ValidateIBCDenom(msg.Token.Denom)
 }
 
-// GetSignBytes implements sdk.Msg
+// GetSignBytes implements sdk.Msg. The function will panic since it is used
+// for amino transaction verification which IBC does not support.
 func (msg MsgTransfer) GetSignBytes() []byte {
-	return sdk.MustSortJSON(ModuleCdc.MustMarshalJSON(&msg))
+	panic("IBC messages do not support amino")
```

Comment:
q: why not return nil bytes? the tx will fail regardless

Reply:
What if someone tries to use this function in an application module? Returning nil would be a silent failure. Also going off this [comment](https://github.com/cosmos/cosmos-sdk/issues/7638#issuecomment-714871518)

Reply:
I guess the logic for panic is that it will clearly communicate why it's failing whereas in Jack's issue he thought it _should_ work.